components/datetime/src/any/datetime.rs

```diff
@@ -262,10 +259,87 @@ impl DateTimeFormatter {
 
         let calendar = AnyCalendar::try_new_unstable(data_provider, kind)?;
 
+        let patterns = PatternSelector::for_options(
+            data_provider,
+            calendar::load_lengths_for_any_calendar_kind(data_provider, locale, kind)?,
+            &locale,
+            &options,
+        )?;
+
+        Ok(Self(
+            raw::DateTimeFormatter::try_new(
+                data_provider,
+                patterns,
+                || calendar::load_symbols_for_any_calendar_kind(data_provider, locale, kind),
+                locale_with_cal,
+                options,
+            )?,
+            calendar,
+        ))
+    }
+
+    // No docs because the "experimental" version is what docs use
```

Comment:
suggestion: worth noting the optional SkeletonsPattern bound in the docs of the other one ("The X bound is only needed with the experimental feature and enables blah blah")